db/migrate/20250207083950_create_tooltips.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTooltips < ActiveRecord::Migration[7.2]
+  def change
+    create_table :tooltips, id: :uuid do |t|
+      t.references :user_account, null: false, foreign_key: true, type: :uuid
```

Comment:
Instead of `foreign_key: true`, can you please update this to comply with Rails 7 best practices:
`foreign_key: { to_table: :user_accounts }`